### PR TITLE
Feature/templates

### PR DESCRIPTION
**Pull request by odrik**
adding main templates

### CODE REVIEW

**Comment by firenaik** [comment]
good for me !!!!

**Comment by Kaboudou** [comment]
C'est bon pour moi

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#CONTRIBUTING
+V2
```
Comment (firenaik) [inline CONTRIBUTING.md]: it's empty !!!!!!!

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1 @@
+PULL_REQUEST_TEMPLATE.md
```
Comment (firenaik) [inline PULL_REQUEST_TEMPLATE.md]: it's empty !!!!!!!

File: CONTRIBUTING.md (hunk 1 above)
Comment (odrik) [reply CONTRIBUTING.md]: oh, shit, let me complete that...

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (odrik) [reply PULL_REQUEST_TEMPLATE.md]: i'll fix it

**Comment by firenaik** [comment]
good for me !!!

**Pull request merged**